jmxtrans-output/jmxtrans-output-elastic/src/test/java/com/googlecode/jmxtrans/model/output/elastic/ElasticWriterTests.java

```diff
@@ -208,11 +208,13 @@ private ElasticWriter createElasticWriter() {
 		Map<String,Object> settings = new HashMap<String,Object>();
 
 		String connectionUrl = "http://localhost";
+        String username      = "elastic";
```

Comment:
It would be nice to add a test that validates that the credentials are sent to elasticsearch, and a test that validates that we can still connect anonymously.